src/components/datePicker/datePicker.tsx

```diff
@@ -0,0 +1,127 @@
+import ReactDatePicker, { registerLocale } from 'react-datepicker';
+import classNames from 'classnames/bind';
+import { FC, ReactNode, useMemo, useRef, forwardRef } from 'react';
+import { be, ru, enGB, es, uk, zhCN, Locale } from 'date-fns/locale';
+import { DatePickerHeader } from './header/datePickerHeader';
+import Input from './input/input';
+import styles from './datePicker.module.scss';
+
+const cx = classNames.bind(styles);
+
+const currentLanguageToLocale: { [key: string]: Locale } = {
+  en: enGB,
+  es,
+  be,
+  ru,
+  uk,
+  zh: zhCN,
+};
+
+const DEFAULT_LOCALE = 'en';
+const DEFAULT_DATE_FORMAT = 'MM-dd-yyyy';
+
+const CustomInput = forwardRef((props: any, ref) => {
```

Comment:
Do we really need two wrappers around `FieldText`?
Can we leave just one or use `FieldText` directly?